sdk/reservations/Azure.ResourceManager.Reservations/src/autorest.md

```diff
@@ -23,6 +23,16 @@ format-by-name-rules:
   'locations': 'azure-location'
   '*Uri': 'Uri'
   '*Uris': 'Uri'
+  "SessionId": "uuid"
+  "DestinationTenantId": "uuid"
+  "BillingScopeId": "arm-id"
+  "SubRequestId": "uuid"
+
+override-operation-name:
+  Reservation_AvailableScopes: GetAvailableScopes
+  CalculateExchange_Post: CalculateReservationExchange
+  Exchange_Post: Exchange
```

Comment:
Is single word ok to this method?

Reply:
Well, I think it's ok because I don't have a better proposal here and we also have single word method name like `Get`, `Update`. Let's use this name and see how the arch board will say.